server/session.go

```diff
@@ -192,9 +197,22 @@ func (apollo *Apollo) retrieveOrganisation() (*core.Organisation, error) {
 		)
 	}
 
+	var parentID *core.OrganisationID
+	if session.Values[sessionOrganisationParent] != nil {
+		pID, ok := session.Values[sessionOrganisationParent].(core.OrganisationID)
+		if !ok {
+			return nil, fmt.Errorf(
+				"invalid organisation parent id stored in session: %v",
+				session.Values[sessionOrganisationParent],
+			)
+		}
```

Comment:
Kunnen we dit niet proberen automatisch oplossen, door de organisation opnieuw op te halen ofzo?